.env.example

```
@@ -1,6 +1,6 @@
 # Backend app metadata
 APP_NAME=Shelfy API
-ENVIRONMENT=development
+ENVIRONMENT=development  # use production in deployed environments
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)